cmd2/parsing.py

```diff
@@ -90,11 +121,11 @@ class StatementParser:
     """
     def __init__(
             self,
-            allow_redirection=True,
-            terminators=None,
-            multiline_commands=None,
-            aliases=None,
-            shortcuts=None,
+            allow_redirection: bool = True,
+            terminators: List[str] = None,
+            multiline_commands: List[str] = None,
+            aliases: Dict[str, str] = None,
+            shortcuts: List[Tuple[str, str]] = None,
     ):
         self.allow_redirection = allow_redirection
         if terminators is None:
```

Comment:
Given a recent bug fix PR #463, it would be beneficial to have a unit test which covers the case when ``terminators`` is ``None``.

Reply:
Agreed. Added.